### ISSUE TITLE
improve the recall of class 1

### ISSUE THREAD

**Issue by Auyin111** [Auyin111/churn-prediction#3]
instead of edit the class weight, try more method to improve it. such as oversampling, under sampling and etc.

**Comment by Auyin111** [comment]
The test set recall are very similar to train set recall report. So the lacking of training data and useful features are the main reasons of bad recall.
